buildroot/share/fonts/genallfont.sh

```diff
@@ -57,7 +57,7 @@ OLDWD=`pwd`
 #
 # Compile the 'genpages' command in-place
 #
-(cd ${DN_EXEC}; gcc -o genpages genpages.c getline.c)
+(cd ${DN_EXEC}; cc -o genpages genpages.c getline.c)
```

Comment:
What it changes?

Reply:
This change was required for a successful build on my macOS 11.1 system.